travis_scripts/install_wiremock.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+WIREMOCK_DOWNLOAD='Smartsheet-WireMock-Bundle.zip'
+WIREMOCK_INSTALL_DIR='Smartsheet-WireMock-Bundle'
+RELEASE_DOWNLOAD_URL='https://github.com/stollgr/wiremock-scenario-templating/releases/download/0.12/Smartsheet-WireMock-Bundle.zip'
```

Comment:
Is there a way to use the github repo that is kicking off the travis run? I don't want the script to be hard coded to my specific repo.